tdex-daemon.md

```diff
@@ -164,6 +179,12 @@ $ tdex genseed
 $ tdex init --seed <generatedSeed> --password <mypassword>
 ```
 
+* **OR** import and restore an existing wallet
```

Comment:
specify that using elements as explorer it will take quite few time to complete the whole rescan

Reply:
and to check the logs of elementsd for something like this:

```
2021-03-04T22:08:43Z [default wallet] RescanFromTime: Rescanning last 122824 blocks
2021-03-04T22:08:43Z [default wallet] Rescan started from block f40b2bfc586ab1f835545454eeb876922fcaf9721c1cfa27b47f1b77f1d06574...
2021-03-04T22:08:58Z [default wallet] AddToWallet daf2ecfb856260d8934a9ed9d0a8ff97a33e7fdf69929f6eefa6ace4a80f8f85  new
2021-03-04T22:09:02Z [default wallet] AddToWallet 501b789754464361da970bf0ddeeb2b04da35b67524021a95083b91d7c630b79  new
2021-03-04T22:09:36Z UpdateTip: new best=5b01987cec905d1dbeeab1228ea5d55f30ab6144ae45f1440b7306ecc5f08d72 height=1218164 version=0x20000000 tx=1940371 date='2021-03-04T22:08:58Z' progress=1.000000 cache=0.0MiB(41txo)
2021-03-04T22:09:43Z [default wallet] Still rescanning. At block 1132718. Progress=0.924000
```